inc/helpers.php

```diff
@@ -572,3 +572,70 @@ function primer_array_replace_recursive( array $array1, array $array2 ) {
 	return (array) $result;
 
 }
+
+/**
+ * Helper function to generate a new menu item object
+ *
+ * @param  string   $title
+ * @param  string   $url
+ * @param  integer  $order
+ * @param  integer  $parent
+ * @param  array    $classes
+ *
+ * @return object
+ *
+ * @since 1.4.2
+ */
+function primer_generate_nav_menu_item( $title, $url, $order, $parent = 0, $classes = array() ) {
```

Comment:
@EvanHerman I don't see this function being used anywhere in the theme. Do we still need it?